cmd/attack/jvm.go

```diff
@@ -146,7 +146,7 @@ func NewJVMStressCommand(dep fx.Option, options *core.JVMCommand) *cobra.Command
 	}
 
 	cmd.Flags().IntVarP(&options.CPUCount, "cpu-count", "", 0, "the CPU core number need to use")
-	cmd.Flags().IntVarP(&options.MemorySize, "mem-size", "", 0, "the memory size need to locate, the unit is MB")
+	cmd.Flags().StringVarP(&options.MemoryType, "mem-type", "", "", "the memory type need to locate, the value can be 'stack' or 'heap'")
```

Comment:
Does it only need to specify the type and not the size here?

Reply:
Yeah, it is used to support OOM, allocate as much memory as possible until an OOM error occurs, so the size is not required.